src/server/api/api.py

```diff
@@ -1,4 +1,11 @@
 from flask import Blueprint
+from flask_cors import CORS
 
-admin_api = Blueprint('admin_api', __name__)
-common_api = Blueprint('common_api', __name__)
+admin_api = Blueprint("admin_api", __name__)
+common_api = Blueprint("common_api", __name__)
```

Comment:
what's the difference between user_api and common_api?
the idea of the common was for routes that will be used by all and admin only for the admin actions.

Reply:
ok actually this makes sense, but maybe the name should be changed to auth_api? since this will be for unauthenticated users? after auth they will get a role and with that either be allowed or not the  other endpoint prefixes? @c-simpson

Reply:
Agreed, seems to make sense to put most under admin_api .

In discussion, agreed that even common_api  stuff needs to require login.

Reply:
My 2c... I like the idea of separating the APIs, maybe we assume all are secure except those in "public" as we will likely have more than 2 controllers.